tasks/sync_repos.py

```diff
@@ -274,8 +274,9 @@ def process_repos(repos):
                 f"{metrics_scope}.sync_repos_using_integration.list_repos_generator"
             ):
                 async for page in git.list_repos_using_installation_generator(username):
-                    received_repos = True
-                    process_repos(page)
+                    if page:
```

Comment:
Small bug fix for if a page never returns any repos; trigger the stuff on line 291.